.github/workflows/build.yml

```diff
@@ -64,7 +64,7 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - name: install system dependencies
-      run: sudo apt-get update && sudo apt-get install libelf-dev mpich libmpich-dev libomp-15-dev libsystemd-dev liburing-dev python3-gi python3-yaml libcap-dev
+      run: sudo apt-get update && sudo apt-get install libelf-dev mpich libmpich-dev libomp-15-dev libsystemd-dev liburing-dev python3-gi python3-yaml libcap-dev libz1g-dev
```

Comment:
Is this also needed in `geopm/.github/workflows/codeql-analysis.yml`?

Reply:
it also needs to be spelled correctly.  Thanks!